addons/account/views/account_report.xml

```diff
@@ -14,6 +14,18 @@
             groups="account.group_account_invoice"
         />
 
+        <report
```

Comment:
add a comment to explain that this report is to print the original pdf(s) that were used to create the vendor bill(s)